Exporter/src/main/java/uk/ac/ebi/biostd/exporter/persistence/dao/SubmissionDao.java

```diff
@@ -54,6 +54,11 @@ public void releaseSubmission(long submissionId) {
         return template.queryForList(queries.getPublicSubmissions(), emptyMap(), String.class);
     }
 
+    public String getPublicSubmissionPathByAccNo(String accNo) {
```

Comment:
`getSubmissionPathByAccNo` as the operation is not related to submission visibility.